tests/tap_tester/base.py

```diff
@@ -362,14 +399,20 @@ def timedelta_formatted(self, dtime, days=0):
 
         except ValueError:
             try:
-                date_stripped = dt.strptime(
-                    dtime, self.BOOKMARK_COMPARISON_FORMAT)
+                date_stripped = dt.strptime(dtime, self.BOOKMARK_COMPARISON_FORMAT)
                 return_date = date_stripped + timedelta(days=days)
 
                 return dt.strftime(return_date, self.BOOKMARK_COMPARISON_FORMAT)
 
             except ValueError:
-                return Exception("Datetime object is not of the format: {}".format(self.START_DATE_FORMAT))
+                return Exception(
+                    "Datetime object is not of the format: {}".format(
+                        self.START_DATE_FORMAT
+                    )
+                )
 
     def is_incremental(self, stream):
-        return self.expected_metadata().get(stream).get(self.REPLICATION_METHOD) == self.INCREMENTAL
+        return (
+            self.expected_metadata().get(stream).get(self.REPLICATION_METHOD)
+            == self.INCREMENTAL
+        )
```

Comment:
Below suggestion looks more readable and is under 100 characters.

```suggestion
        return (
            self.expected_metadata().get(stream).get(self.REPLICATION_METHOD) == self.INCREMENTAL
        )
```

Reply:
Done